src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/XmlReaderWrapper.cs

```diff
@@ -319,7 +323,7 @@ public override Task<string> ReadInnerXmlAsync()
 
     internal static class XmlReaderExtensions
     {
-        static private uint IsTextualNodeBitmap = 0x6018; // 00 0110 0000 0001 1000
+        static private uint s_isTextualNodeBitmap = 0x6018; // 00 0110 0000 0001 1000
```

Comment:
I realize this is pre-existing, but looks like this should be a const rather than a static.